tests/antares/services/api_services/test_study_api.py

```diff
@@ -648,6 +657,53 @@ def test_output_get_matrix(self):
             assert isinstance(matrix, pd.DataFrame)
             assert matrix.equals(expected_matrix)
 
+    def test_output_get_mc_all(self):
+        self.study._areas["area_test"] = self.area
+        self.study._areas["area_test_1"] = self.area_1
+        self.study._outputs["test-output-link"] = self.output_link
+        self.study._outputs["test-output-area"] = self.output_area
+
+        frequency = Frequency(Frequency.ANNUAL.value)
+        with requests_mock.Mocker() as mocker:
+            matrix_link_url = f"https://antares.com/api/v1/studies/{self.study_id}/raw?path=output/{self.output_link.name}/economy/mc-all/links/{self.area.id}/{self.area_1.id}/values-{frequency.value}"
```

Comment:
As briefly discussed, I think we could instead use the endpoint `original-file` in the web implementation, so that we can use the same parsing method in the web implementation as in the local implementation.
(its uses the `read_csv` method of pandas the appropriate parameters in particular to read multiple lines for the column headers)